src/mapbox/CountyLayer.js

```diff
@@ -6,26 +6,48 @@ import {county, selectedCounty, hoverCounty} from './LayerStyles';
 import {DataContext} from '../App'
 import {
   getCountyAndColorDictionary,
-  retrieveCountyAndMetricDictionary,
 } from "./CountyColorsUtil";
 
+import {extractCountyAndMetricDictionary,getDataForSelector} from "./ExtractCountyDataUtil.js"
+import {
+    getExtraDataLabelDictionary,
+} from './DataSelectionUtil';
+
 const CountyLevel = () => {
   /**
    * counties = County GeoJSON and county level data.
    * filter = hovered/highlight zipcode/county
    */
   const filters = useSelector(state => state.filters)
-  const { countyData, counties } = useContext(DataContext)
+
+  const { countyData, counties, metaData } = useContext(DataContext)
 
   const selectedFeat = useSelector(state => state.selectedFeat)
   const extraDataFeat = useSelector(state => state.extraDataMenuFeat)
```

Comment:
@KhalfaniW I tracked the bug down to triggering when this is `undefined`

Without either knowing Redux or how the data is set up I have no idea where to look at why it's undefined. Or how to define it. Any thoughts?